sbbbattlesim/characters/SBB_CHARACTER_CERBERUS.py

```diff
@@ -7,17 +7,19 @@
 
 logger = logging.getLogger(__name__)
 
+
 class GrimSoulOnDeath(OnDeath):
     last_breath = True
 
-    def handle(self, stack, *args, **kwargs):
-        targets = self.manager.player.valid_characters(lambda char: char.slay and char.id not in ['SBB_CHARACTER_CERBERUS', 'SBB_CHARACTER_LOBO'])
```

Comment:
@Reggles44 why did you remove southern siren? Its hard coded to never be triggered by grim soul (even if it has riverwish)